pkg/controller/pravega/pravega_controller_test.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/pravega/pravega-operator/pkg/controller/pravega"
 
 	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
v1 has already been imported by the name corev1 in the previous line

Reply:
fixed